Resources/public/css/theme/views/contenteditform.css

```diff
@@ -24,3 +24,22 @@
 .ez-view-contenteditformview .is-collapsed .fieldgroup-name:after {
     content: "\E604";
 }
+
+.ez-view-contenteditformview .ez-editfield-not-translatable-msg {
```

Comment:
this should not be in contenteditform.css but in fieldedit.css as you actually style a field edit view